API/Features/Customer/Requests/GetPortalUsers.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using DLCS.Model.Customers;
+using DLCS.Repository;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.Features.Customer.Requests;
+
+public class GetPortalUsers : IRequest<IList<User>>
+{
+    public int CustomerId { get; set; }
+}
+
+public class GetPortalUsersHandler : IRequestHandler<GetPortalUsers, IList<User>>
+{
+    private readonly DlcsContext dbContext;
+
+    public GetPortalUsersHandler(DlcsContext dbContext)
+    {
+        this.dbContext = dbContext;
+    }
+
+    public async Task<IList<User>> Handle(GetPortalUsers request, CancellationToken cancellationToken)
+    {
+        var users = await dbContext.Users.Where(u => u.Customer == request.CustomerId)
+            .ToListAsync(cancellationToken);
```

Comment:
Use `AsNoTracking()`.

Another thought, for later - handlers like this would be a good candidate for the new `IStreamRequest<T>` handler in [Mediatr 10](https://github.com/jbogard/MediatR/releases/tag/v10.0.0), which supports returning `IAsyncEnumerable<T>`. Would mean using `.AsAsyncEnumerable()` rather `.ToListAsync()`.